src/content/test/semantics/headers-attribute.tsx

```diff
@@ -0,0 +1,117 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { create, React } from '../../common';
+
+export const infoAndExamples = create(({ Markup, Link }) => (
+    <>
+        <h1>Headers attribute</h1>
+        <p>
+            The <Markup.Term>headers</Markup.Term> attribute of a <Markup.Term>{`<td>`}</Markup.Term> element must reference the correct{' '}
+            <Markup.Term>{`<th>`}</Markup.Term> element.
+        </p>
+
+        <h2>Why it matters</h2>
+        <p>
+            When assistive technologies describe data tables, they mention both the data cells and their programmatically-related header
+            cells. When incorrect header cells are reported, users are likely to find the table confusing.
+        </p>
+
+        <h2>How to fix</h2>
+        <ul>
+            <li>
+                Good: Modify the <Markup.Term>headers</Markup.Term> attribute of the <Markup.Term>{`<td>`}</Markup.Term> element to match
+                the <Markup.Term>id</Markup.Term> attribute of the correct <Markup.Term>{`<th>`}</Markup.Term> element, or
+            </li>
+            <li>
+                Better: Eliminate multi-level headings and define headers using <Markup.Term>scope</Markup.Term> attributes.
+            </li>
+        </ul>
+        <p />
+
+        <h2>Example</h2>
+        <Markup.PassFail
+            failText={
+                <p>
+                    This complex data table has two levels of headers. Three of the data cells' <Markup.Term>headers</Markup.Term>{' '}
+                    attributes should refer to both levels, but incorrectly refer to only the top-level header. Assistive technologies will
+                    announce only "Staff" as the header for those data cells. Some users will find it difficult to understand the table.
```

Comment:
In the passing example, the "Staff" header text was replaced by "Faculty"; the failing example should be consistent in the text label and here in this text

Reply:
Good catch! Checking with Fer the right word to use here.

Reply:
Confirmed that they should both be Staff.